### ISSUE TITLE
MissingCompilationDirectory error

### ISSUE THREAD

**Issue by kripken** [yurydelendik/wtmaps-utils#4]
I think this happened after I updated LLVM, but I updated a few other things too so I'm not sure. STR:

```bash
$ wdwarf-cp emcc-6-pre-byn.wasm -o final.wasm -m emcc-9-output_map.map -w emcc-8-wasm-opt.wasm 
thread 'main' panicked at 'new dwarf: MissingCompilationDirectory', src/libcore/result.rs:1165:5
```

[emscripten_temp.zip](https://github.com/yurydelendik/wtmaps-utils/files/3881563/emscripten_temp.zip)

**Comment by kripken** [comment]
Oh, this seems to happen when building with `-gline-tables-only`. So maybe that isn't a supported mode with wtmaps-utils?

**Comment by yurydelendik** [comment]
> this seems to happen when building with -gline-tables-only

I'll check this anyway, just to see what's going on

**Comment by yurydelendik** [comment]
Looks like it was fixed by https://github.com/gimli-rs/gimli/pull/430 -- updating to new gimli with fix the issue.

**Comment by yurydelendik** [comment]
Fixed by 011e955d01dafed17c892
